ports/glew/vcpkg.json

```
@@ -0,0 +1,13 @@
+{
+  "name": "glew",
+  "version-string": "2.1.0",
+  "port-version": 11,
+  "description": "The OpenGL Extension Wrangler Library (GLEW) is a cross-platform open-source C/C++ extension loading library.",
+  "homepage": "https://github.com/nigels-com/glew",
+  "dependencies": [
+    {
+      "name": "opengl",
+      "platform": "windows"
+    }
```

```suggestion
    "opengl"
```

The port `opengl` is intended to represent `find_package(OpenGL)`, so that is a true dependency on all platforms.

---

Updated, thanks for your review!